exo-go/src/app_runner/index.go

```diff
@@ -0,0 +1,100 @@
+package appRunner
+
+import (
+	"fmt"
+	"math"
+	"path"
+	"sync"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/docker_compose"
+	"github.com/Originate/exosphere/exo-go/src/logger"
+	"github.com/Originate/exosphere/exo-go/src/service_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/types"
+	"github.com/fatih/color"
+)
+
+// AppRunner runs the overall application
+type AppRunner struct {
+	AppConfig            types.AppConfig
+	Logger               *logger.Logger
+	Env                  map[string]string
+	DockerConfigLocation string
+	OnlineTexts          map[string]string
+}
+
+// NewAppRunner is AppRunner's constructor
+func NewAppRunner(appConfig types.AppConfig, logger *logger.Logger, cwd string) *AppRunner {
+	appRunner := &AppRunner{AppConfig: appConfig, Logger: logger, Env: appConfigHelpers.GetEnvironmentVariables(appConfig), DockerConfigLocation: path.Join(cwd, "tmp")}
```

Comment:
Thoughts on styling thins in the following way to make it a little easier to read:
```go
appRunner := &AppRunner{
  AppConfig: appConfig, 
  Logger: logger, 
  Env: appConfigHelpers.GetEnvironmentVariables(appConfig), 
  DockerConfigLocation: path.Join(cwd, "tmp"),
}
```